content/documentation.rst

```diff
@@ -0,0 +1,110 @@
+Documentation
+#############
+
+:menu_order: 101
+
+.. contents::
+
+
+General
+=======
+
+General introduction about why we want documentation, how the documentation
+is presented
+
+talk about different documentation formats and different audiance
+
+
+From /generic.html talk about newliens and how we manage diffs .. move that
+part here.
+
+For narrative documentation use semantic newlines.
+Make lines short, and break lines at natural places,
+such as after commas and semicolons, rather than after the Nth column.
```

Comment:
@adiroiban I wonder why? Is documentation more readable with the semantic newlines?